### PR TITLE
Remove references to LGTM in code

### PR DESCRIPTION
**Pull request by calumgrant**
Remove a few references to LGTM in various places.

Addresses https://github.com/github/code-scanning/issues/7229 and https://github.com/github/codeql-python-team/issues/610

### CODE REVIEW

File: python/ql/test/3/library-tests/PointsTo/typehints/test.py [hunk 1]
```diff
@@ -8,11 +8,3 @@ def bar(s:set)->Set:
...
 t1 = Optional[Optional[int]]
 t2 = Optional[int][int]
-
-# ODASA-8075
-# Commented out until the fix has been pushed to LGTM.com
-#class baz():
-#    pass
-#
-#while True:
-#    baz = baz[baz]
```
Comment (aibaars) [inline python/ql/test/3/library-tests/PointsTo/typehints/test.py:18]: I'd hope the fix has been pushed to LGTM.com by now, so perhaps we should uncomment the code again.
```suggestion
class baz():
    pass

while True:
    baz = baz[baz]
```

**Review by aibaars — COMMENTED** [review]
Looks fine to me.

File: python/ql/test/3/library-tests/PointsTo/typehints/test.py (hunk 1 above)
Comment (calumgrant) [reply python/ql/test/3/library-tests/PointsTo/typehints/test.py:18]: 👍  I've put back the test and updated the test results.

**Comment by calumgrant** [comment]
Addressed the feedback so far, and the tests pass.

@yoff, I reenabled a points-to test although I'm not 100% sure it's needed.

**Review by erik-krogh — APPROVED** [review]
JS 👍

**Review by aibaars — APPROVED** [review]

File: python/ql/test/3/library-tests/PointsTo/typehints/test.py (hunk 1 above)
Comment (tausbn) [reply python/ql/test/3/library-tests/PointsTo/typehints/test.py:18]: Oh yeah, I fixed the underlying problem ages ago. Good catch, @aibaars. 🙂

**Review by tausbn — APPROVED** [review]
Python bits look good to me. 👍

**Pull request merged**